recirq/quantum_chess/quantum_board_test.py

```diff
@@ -67,6 +67,7 @@ def test_initial_state(board):
         assert probs[bit] == 0.0
 
 
+        
```

Comment:
I think the standards are two blank lines in between top-level functions.  We don't need a third line.